inc/saiport.h

```diff
@@ -2528,6 +2528,22 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_JSON_FORMATTED_DEBUG_DATA_SIZE,
 
+    /**
+     * @brief Query MAC local fault status
+     *
+     * @type bool
+     * @flags READ_ONLY
+     */
+    SAI_PORT_ATTR_MAC_LOCAL_FAULT_STATUS,
```

Comment:
is it better to add this as PORT_ATTR_FAULT_STATUS and then define enum such as mac_local_faluse, remote fault, and in the future could be other types of fault.